RELEASE_NOTE_DRAFT.md

```diff
@@ -1,25 +1,112 @@
 11.0.0
 ====================
 
-This is the 11.0.0 major release. 
+This is the 11.0.0 major release.
+
+Central theme of this release is addition of new APIs, features as requested by the community and decrease Tech Debt.
+This is our yearly major release.
```

Comment:
The 10.0.0 release was in July, 2019. So this is the first major release in over two years.

Reply:
Removed it.